DXRMapFixups/DeusEx/Classes/DXRFixupM06.uc

```diff
@@ -756,6 +756,12 @@ function AnyEntryMapFixes()
         }
         HandleJohnSmithDeath();
         SetTimer(1.0, True); //To handle updating the DTS goal description
+
+        if (dxr.flagbase.GetBool('Have_ROM')) {
```

Comment:
not sure this super specific bingo stuff belongs in DXRMapFixups, I think it might be time to split up DXREvents.uc by mission as I said in my other comment